Section3/Exercise7/Main7.cpp

```diff
@@ -0,0 +1,92 @@
+/*Exercise 7 - Lottery
+Write a program that will simulate the lottery.Create two arrays of 5 integers.The first array should be populated with 5 random numbers from 1 - 20. For the second array, ask the user to input 5 numbers from 1 - 20. Make sure to handle invalid input and make sure numbers are not repeated.
+For each number that is matched in the same index, give the user $10.*/
+#include<iostream>
+#include <cstdlib>    
+#include <ctime>  
+using namespace std;
+bool repeatValues(int arrRepeat[], int size) {
+	for (int pos = 1; pos < size; pos++)
+	{
+		for (int posL = 0; posL < pos; posL++)
+		{
+			if (arrRepeat[pos] == arrRepeat[posL])
+			{
+				return true;
+			}
+		}
+	}
+	return false;
+}
+int main()
+{
+	int ArrayLotery[5];
+	int ArrayUser[5];
+	int ArrLoterySize = sizeof(ArrayLotery) / sizeof(ArrayLotery[0]);
+	int ArrUserSize = sizeof(ArrayUser) / sizeof(ArrayUser[0]);
+
+	cout << "Welcome to the lottery!" << endl;
+
+
+	int UserValues;
+
+	for (int UserSpace = 0; UserSpace < ArrUserSize; UserSpace++)
+	{
+		int entry = UserSpace + 1;
+		cout << "Please enter your entry number #" << entry << "(1-20)" << endl;
+		cin >> UserValues;
+		for (int pruebaU = 0; pruebaU < ArrUserSize; pruebaU++)
+		{
+			if (ArrayUser[pruebaU] == UserValues) {
+				cout << "Sorry equal number" << endl;
+				cin.clear();
+				cin.ignore(1000, '\n');
+				UserSpace -= 1;
+			}
+		}
+
+		if ((!cin.good()) || UserValues < 1 || UserValues>20) {
+			cout << "Sorry, invalid Input! or equal number  " << endl;;
+			cin.clear();
+			cin.ignore(1000, '\n');
+			UserSpace -= 1;
+		}
+		ArrayUser[UserSpace] = UserValues;
+	}
+
+	srand(time(0));
+
+	do {
+		for (int Result = 0; Result < ArrLoterySize; Result++)
+		{
+			int LoteryValue = (rand() % 20) + 1;
+			ArrayLotery[Result] = LoteryValue;
+		}
+
+	} while (repeatValues(ArrayLotery, ArrLoterySize));
+	cout << "Lottery results: ";
+	for (int L = 0; L < ArrLoterySize; L++)
+	{
+		cout << ArrayLotery[L] << ",";
+	}
+
+	int Profits=0;
+	if (ArrayLotery[0] == ArrayUser[0]) {
```

Comment:
What if I get two of the numbers correct? I would only get 10 dollars in that case. I can never get more than 10 dollars.

Reply:
the user only wins if he has the same numbers in the same boxes of the array, which is why I did it with several if's and
so I control that I can win more than 1
![Ifs](https://user-images.githubusercontent.com/36427649/179555104-ef0c550d-ac9d-43ec-80cb-7b5c1c6fbed0.PNG)
0